src/libiotc/event_loop/iotc_event_loop.c

```diff
@@ -35,7 +36,7 @@ static size_t iotc_bsp_event_loop_count_all_sockets(
 
   uint8_t evtd_id = 0;
   for (; evtd_id < num_evtds; ++evtd_id) {
-    const iotc_evtd_instance_t* event_dispatcher = event_dispatchers[evtd_id];
+    const iotc_evtd_instance_t *event_dispatcher = event_dispatchers[evtd_id];
```

Comment:
probably inadvertently wrong code formatter was unleashed on this code. * should stick to type.